ngraph/frontend/tensorflow/src/ngraph_conversions.cpp

```diff
@@ -10,27 +10,25 @@ namespace ov {
 namespace frontend {
 namespace tf {
 
-void NHWCtoNCHW(const std::string& op_name, bool need_convert, ov::Output<ov::Node>& node) {
+void convert_nhwc_to_nchw(const std::string& op_name, bool need_convert, ov::Output<ov::Node>& node) {
     if (need_convert) {
         auto rank = node.get_shape().size();
         if (rank == 4) {
             Transpose<0, 3, 1, 2>(node);
         } else if (rank == 5) {
             Transpose3D<0, 4, 1, 2, 3>(node);
         }
-        SetTracingInfo(op_name, node);
     }
 }
 
-void NCHWtoNHWC(const std::string& op_name, bool need_convert, ov::Output<ov::Node>& node) {
+void convert_nchw_to_nhwc(const std::string& op_name, bool need_convert, ov::Output<ov::Node>& node) {
     if (need_convert) {
         auto rank = node.get_shape().size();
         if (rank == 4) {
             Transpose<0, 2, 3, 1>(node);
         } else if (rank == 5) {
             Transpose3D<0, 2, 3, 4, 1>(node);
         }
```

Comment:
please add FRONTEND_CHECK that rank < 5

Reply:
not needed, just do nothing for rank < 4